src/core/core.interaction.js

```diff
@@ -41,39 +41,38 @@ function evaluateAllVisibleItems(chart, handler) {
 	}
 }
 
+function useBinarySearch(axis, controller, _sorted, intersect) {
```

Comment:
It's not quite clear from the name that it's deciding whether to use a binary search and not actually doing the binary search. How about `shouldUseBinarySearch` or `binarySearchEnabled`?